nuxt.config.js

```diff
@@ -35,7 +37,9 @@ export default {
   ],
 
   'nuxtjs-drupal-ce': {
-    baseURL: 'http://example.d9playground.localdev.space'
+    baseURL: DRUPAL_BASE_URL,
```

Comment:
the module should automatically apply DRUPAL_BASE_URL even without your code, it already has that logic in place. You can you try just setting the baseurl?

Reply:
I tried, but it was not working so i fixed it here - it gets explicitly overwritten in line186 plugin.js `options.baseURL = runtimeConfig.baseURL || options.baseURL` - runtimeConfig.baseURL takes precedence

Reply:
ok, so let's open a bug report at the module then and fix it separately?